e2e/cypress/integration/homepage.js

```diff
@@ -1,6 +1,87 @@
 describe('Ursus Homepage', () => {
```

Comment:
I think it makes sense to add the static page tests to this file, but we should rename the file and change the 'describe' statement